src/features/forest/components/Tree.tsx

```diff
@@ -247,18 +260,6 @@ export const Tree: React.FC<Props> = ({ treeIndex }) => {
         </>
       )}
 
-      <div
```

Comment:
It seems you have reverted a change from main. This is the new health bar that shows on a tree.

Consider rebasing onto main to avoid removing this feature.

Reply:
I did not see that one my bad gonna update it right now !